react/javascript/demo-external-api/src/components/Sheets.jsx

```diff
@@ -0,0 +1,192 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2021 Looker Data Sciences, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+import React, { useEffect, useState, useContext } from 'react'
+import PropTypes from 'prop-types'
+import {
+  ButtonOutline,
+  Heading,
+  Code,
+  Space,
+  Tooltip,
+  useConfirm,
+} from '@looker/components'
+import { ExtensionContext2 } from '@looker/extension-sdk-react'
+import { SheetsContext } from './SheetsProvider'
+import { SheetsTable } from './SheetsTable'
+import { SheetsForm } from './SheetsForm'
+
+/**
+ * This is the id of a sample sheet provided by Google. It is used to create
+ * an initial spreadsheet to edit. A new spreadsheet can be created at any
+ * time by pressing the recreate sheet from source button.
+ */
+const sourceSpreadsheetId = '1BxiMVs0XRA5nFMdKvBdBZjgmUUqptlbs74OgvE2upms'
```

Comment:
Are we sure that anyone can access this, or will non @google.com folks get an error?

Reply:
Perhaps just something to call out to change in the readme, if we can't share globally... Will also need to change the range declaration

Reply:
Either way maybe something for the default `.env` config.

Reply:
I've added a comment to the readme for now.